src/main/java/com/dotcms/saml/service/handler/AuthenticationResolverHandlerFactory.java

```diff
@@ -52,6 +52,8 @@ public AuthenticationHandler getAuthenticationHandlerForSite(final IdentityProvi
 				return new HttpRedirectAuthenticationHandler(this.samlCoreService, this.messageObserver, this.samlConfigurationService);
 			case DotSamlConstants.DOTCMS_SAML_AUTHN_PROTOCOL_BINDING_POST:
 				return new HttpPOSTAuthenticationHandler(this.samlCoreService, this.velocityEngine, this.messageObserver);
+			case "Http-POST-Raw":  // todo: change this on new core change
```

Comment:
Why not adding this String to the `DotSamlConstants` class?

Reply:
it will, but we need a new dotCMS version for it, 21.03 or 21.04